src/program.js

```diff
@@ -298,6 +298,11 @@ Example: $0 --help run.
         type: 'string',
         coerce: coerceCLICustomPreference,
       },
+      'jsconsole': {
+        describe: 'Open web browser console.',
```

Comment:
Let's make it clear that this is a shortcut for something one could do manually. How about this description:

    Open Tools > Web Developer > Browser Console on Firefox startup